contrib/intel/jenkins/tests.py

```diff
@@ -60,6 +60,8 @@ def cmd(self):
     def options(self):
         if (self.util_prov):
             opts  = "-f {} -p {};{}".format(self.fabric, self.core_prov, self.util_prov)
+        elif (self.core_prov == 'psm3'):
+            opts = "-p {}".format(self.core_prov)
```

Comment:
I can't remember, did you already try changing the fabric mapping to ib0 instead of mlx5 so it could pass on both psm3 and verbs?

Reply:
I talked to Sean and Nikhil and they were not in favor of changing the mapping  so I added this small change for psm3.